### PR TITLE
[DOC] Add links to source code

### PR DESCRIPTION
**Pull request by harupy**
## What changes are proposed in this pull request?

Add links to source code in the documentation.

jump-demo

## How is this patch tested?

Manually verified the links work properly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/theme/mlflow/static/css/theme.css [hunk 1]
```diff
@@ -5340,3 +5340,11 @@ a .rst-content code {
...
 }
 /*# sourceMappingURL=theme.css.map */
+
+div.viewcode-block:target {
+    background-color: #f4debf;
+    border-top: 1px solid #ac9;
+    border-bottom: 1px solid #ac9;
+    margin: 0px -12px;
+    padding: 0px 12px;
+}
```
Comment (harupy) [inline docs/theme/mlflow/static/css/theme.css]: Screen Shot 2020-03-24 at 12 24 35

**Comment by mparkhe** [comment]
This is a fantastic change. I clicked through the documentation and this will be super helpful. 

One question, I had was can we change the colors for [source] link and the highlighted (clicked) function/class? The green/orange colors look different from the rest of the webpage. @gioa what colors do you recommend?

**Comment by harupy** [comment]
@mparkhe Thanks for the comment.

> Can we change the colors for [source] link and the highlighted (clicked) function/class?

Yes, we can change them.

**Comment by harupy** [comment]
Now it looks like this. @gioa What do you think?

Screen Shot 2020-04-12 at 19 15 03

Screen Shot 2020-04-12 at 19 13 54

**Comment by aarondav** [comment]
Neat. It would be even cooler if it linked to the GitHub source, but this is cool as-is!

**Review by aarondav — APPROVED** [review]

**Pull request merged**